### PR TITLE
Export font information for font loader js

### PR DESCRIPTION
**Pull request by adamszalapski**
## Jira

http://vjira2:8080/browse/BDS-212

## Summary

MOve from manual adding a critical font to using generated JSON file.

## Details

This is the first step to move to use generated file. Now we move those file in the critical-font component but in the final result, we want this file will be generated automatically or use file generated via webpack but running as pre webpack build.

## How to test

Run this code locally.
Check if the website is run correctly.

### CODE REVIEW

**Comment by sghoweri** [comment]
@adamszalapski are these Jest tests stalling locally for you or is this just an issue on Travis?

https://travis-ci.com/boltdesignsystem/bolt/jobs/240704435

**Comment by adamszalapski** [comment]
@danielamorse, @sghoweri I add the missing file, sorry for that. Right now should be all right.

File: packages/build-tools/create-webpack-config.js [hunk 1]
```diff
@@ -166,9 +165,7 @@ async function createWebpackConfig(buildConfig) {
       });
 
       const useHotMiddleware =
-        Array.isArray(fullBuildConfig.lang) && fullBuildConfig.lang.length > 1
-          ? false
-          : true;
+        Array.isArray(fullBuildConfig.lang) && fullBuildConfig.lang.length > 1;
```
[the remark this answers is not in the document]
Comment (danielamorse) [reply packages/build-tools/create-webpack-config.js]: @adamszalapski I see you requested another review yesterday. Did you see this comment^?

[the remark this answers is not in the document]
Comment (adamszalapski) [reply packages/build-tools/create-webpack-config.js]: Good catch @danielamorse i only simplified those conditions and miss that.

[the remark this answers is not in the document]
Comment (adamszalapski) [reply packages/build-tools/create-webpack-config.js]: Fixed

**Comment by danielamorse** [comment]
Thanks @adamszalapski!

I saw the failing tests, pulled down the latest and ran `npm run lint` and I see the errors locally, too:
```
> bolt@0.0.0-development lint:scss /Users/morsd/sites/bolt
> stylelint 'packages/**/*.scss' 'docs-site/**/*.scss' --config .stylelintrc


/Users/morsd/sites/bolt/packages/components/bolt-icons/src/index.js
    2:15  error  Unable to resolve path to module './icons/academy.min'                        import/no-unresolved
    4:15  error  Unable to resolve path to module './icons/add-open.min'                       import/no-unresolved
...
```

Any idea what's happening here?

**Comment by adamszalapski** [comment]
> Thanks @adamszalapski!
> 
> I saw the failing tests, pulled down the latest and ran `npm run lint` and I see the errors locally, too:
> 
> ```
> > bolt@0.0.0-development lint:scss /Users/morsd/sites/bolt
> > stylelint 'packages/**/*.scss' 'docs-site/**/*.scss' --config .stylelintrc
> 
> 
> /Users/morsd/sites/bolt/packages/components/bolt-icons/src/index.js
>     2:15  error  Unable to resolve path to module './icons/academy.min'                        import/no-unresolved
>     4:15  error  Unable to resolve path to module './icons/add-open.min'                       import/no-unresolved
> ...
> ```
> 
> Any idea what's happening here?

That was my fault because i push icons related files to this PR but right now is fixed.

**Comment by sghoweri** [comment]
PR was released with v2.14.0

**Pull request merged**